### ISSUE TITLE
Kong Ingress Controller 0.4.0 released

### ISSUE THREAD

**Issue by hbagdi** [ollystephens/acme-kong-kube-helper#5]
Hello,

To tackle this problem, Kong Ingress Controller sets `preserve_host` to `true` by default and hence thing should work.
Could you try to use that and update this repository?

**Comment by ollystephens** [comment]
I've not had a chance to try it myself yet (will do soon) but I've added a note to the README saying it should no longer be needed. I will update it again with a more definitive statement once I've verified.

Thanks for improving the ingress controller!

**Comment by C45tr0** [comment]
Sorry to comment on a closed issue, but 0.4.0 fixes one issue and creates a new one.

It also changed the default of strip_path: true. Which caused the same routing issues. I made a fix for this and then remembered I was using kong in db less mode. 

StripPath: https://github.com/C45tr0/acme-kong-kube-helper/commit/f6854bb5f70c62e982543d8b12883378ac917d83
DB Less(partially #3): https://github.com/C45tr0/acme-kong-kube-helper/commit/a048f3fa5268ec5117019a0946db08f2447dac33

For the db less, I also needed to add permissions and a default KongIngress. This may not be the perfect option. Was just trying to get something working. 

`kubectl patch clusterrole kong-ingress-kong --type='json' -p='[{"op": "add", "path": "/rules/0", "value": {"apiGroups": ["extensions"], "resources": ["ingresses"], "verbs": ["update"]}}]'`

```
apiVersion: configuration.konghq.com/v1
kind: KongIngress
metadata:
  name: cert-manager-http01
route:
  strip_path: false
  preserve_host: true
```
